req-travis-conda.txt

```diff
@@ -10,3 +10,5 @@ pyzmq==14.7.0
 scipy==0.16.0
 six==1.9.0
 toolz==0.7.4
+Cython==0.23.4
```

Comment:
It would, but it would be installed with `pip`, putting it here makes sure that on Travis the Conda binaries are installed instead.